accounts/ui/tests/run-acceptance-test.sh

```diff
@@ -29,7 +29,7 @@ trap clean_up SIGHUP SIGINT SIGTERM
 if [ -z "$TEST_INFRA_DIRECTORY" ]
 then
 	cleanup=true
-	testFolder=$(< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+	testFolder=$(mktemp -d)
 	printf "creating folder $testFolder for Test infrastructure setup\n\n"
 	export TEST_INFRA_DIRECTORY=$testFolder
```

Comment:
this should be `$testFolder/tests` since we are now copying to a temp dir that's already created.